_includes/highlights.html

```diff
@@ -9,48 +9,40 @@
+        <div class="usa-media-block__img">{% asset accessibility-sme alt="Accessibility SME icon" %}</div>
 
         <div class="usa-media-block__body">
-          <h3 class="usa-graphic-list__heading"><a href="{{ site.baseurl }}/accessibility-expert/">Accessibility Subject Matter Expert (SME)</a></h3>
+          <h3 class="usa-graphic-list__heading"><a href="{{ site.baseurl }}/accessibility-expert/">Accessibility Subject-Matter Expert (SME)</a></h3>
```

Comment:
I'm opposed to the "-" 

I don't know if it is right. Generally I trust, Wikipedia, but it us used more without a "-" in this article than it is with:
https://en.wikipedia.org/wiki/Subject-matter_expert 

Ultimately though, the GSA doesn't seem to use the "-" https://www.google.com/search?q=site%3Agsa.gov+subject+matter+expert

Reply:
Good points. I removed the hyphen.